apk_iman_ba/lib/Pages/searchpage.dart

```diff
@@ -233,7 +233,7 @@ class _SearchPageState extends State<SearchPage> {
                             textAlign: TextAlign.center,
                           ),
                           Image.asset(
-                            "lib/Images/Compass.png",
+                            "assets/images/Compass.png",
```

Comment:
Pokusaj da ti i sami nazivi fajlova slika pocinju malim slovom.
```suggestion
                            "assets/images/compass.png",
```